src/backend/src/controllers/users.controllers.ts

```diff
@@ -41,14 +42,19 @@ export const getUserSettings = async (req: any, res: any) => {
 
 export const updateUserSettings = async (req: any, res: any) => {
   const userId: number = parseInt(req.params.userId);
-  if (!req.body || !req.body.defaultTheme) {
-    return res.status(404).json({ message: 'No settings found to update.' });
+  if (!userId) {
+    return res.status(404).json({ message: `could not find valid userId` });
+  }
```

Comment:
lets also add a check here for getting hte user with the given userId from prisma, then if they dont exist that should be a 404 too